distribution/pom.xml

```diff
@@ -626,7 +626,7 @@
                                         <argument>-c</argument>
                                         <argument>io.confluent.druid.extensions:confluent-extensions</argument>
                                         <argument>-c</argument>
-                                        <argument>org.apache.druid.extensions.contrib:opentelemetry-emitter</argument>
+                                        <argument>org.apache.druid.extensions.contrib:druid-opentelemetry-extensions</argument>
```

Comment:
it looks like you're moving the existing code to a different extension. I don't think we can do that since that would be a breaking change for existing deployment. I think we should keep it part of the opencensus-extensions for now and then provide a migration path.

Reply:
> those are two different extensions, why would we be removing the existing one?

The Otel emitter extension is already added as an argument in [L623](https://github.com/confluentinc/druid/blob/24.0.2-confluent/distribution/pom.xml#L623).  I just removed the duplicate entry. 

> it looks like you're moving the existing code to a different extension.

My understanding is that the existing behavior hasn't changed.

The Otel code is already defined as a separate [extension](https://github.com/confluentinc/druid/blob/24.0.2-confluent/extensions-contrib/opentelemetry-extensions/src/main/resources/META-INF/services/org.apache.druid.initialization.DruidModule). The code hasn't moved.

The OC extension has a [dependency](https://github.com/confluentinc/druid/blob/24.0.2-confluent/extensions-contrib/opencensus-extensions/pom.xml#L68) on the Otel extension module. I verified from the build files, that the OC extensions folder still contains the Otel extensions jar. Mentioning just the OC extension in the `druid.extensions.loadList` still loads the OTel extension.

```
❯ pwd
/Users/knarayan/localSetup/apache-druid-24.0.2/extensions/druid-opencensus-extensions
❯ ls
druid-opencensus-extensions-24.0.2.jar     grpc-core-1.14.0.jar                       gson-2.7.jar                               opencensus-proto-0.2.0.jar
druid-opentelemetry-extensions-24.0.2.jar  grpc-protobuf-1.14.0.jar                   jsr305-2.0.1.jar                           opentelemetry-proto-0.19.0-alpha.jar
error_prone_annotations-2.11.0.jar         grpc-protobuf-lite-1.14.0.jar              opencensus-api-0.12.3.jar                  proto-google-common-protos-1.0.0.jar
grpc-context-1.14.0.jar                    grpc-stub-1.14.0.jar                       opencensus-contrib-grpc-metrics-0.12.3.jar protobuf-java-3.19.2.jar
```


The `<argument>` was added here so that, in addition to the existing behaviour, the resulting distribution would enable us to load Otel extension just by itself.  If this is not advisable,  I'll remove it.

Reply:
makes sense, thanks for clarifying